.concourse/pipeline.yaml.gomplate

```diff
@@ -2035,13 +2039,13 @@ jobs:
     - {{ . -}}{{ end }}
     {{- end }}
     params:
-      file: output/kubecf-v*.tgz
+      file: output/kubecf-[0-9]*.tgz
   - put: s3.kubecf-bundle
     {{- if $config.workertags }}
     tags: {{ range $config.workertags }}
     - {{ . -}}{{ end }}
     {{- end }}
     params:
-      file: output/kubecf-bundle-v*.tgz
+      file: output/kubecf-bundle-*.tgz
```

Comment:
Is there a reason why the number glob is missing here?

Reply:
not really, it's just not needed. It will shell-expand to the number of the semver anyways, we don't have another file like kubecf-bundle-foo.tgz.